notebooks/Liquidators_analysis.ipynb

```diff
@@ -0,0 +1,1351 @@
ChQupXr16fHn01q1bafLkyUwCwfr6+nT27FlydHSk8vJykkgkdOrUKfLy8iKiyve8SZMmTIJylpaWtGXLFmrevDk9ffqUjI2Nad++fRQYGEhERIcOHaLo6Gi6ceNGrWsTEanWCOArdOjQIXVfAo0ePZrq1q1bzQeioqKC1q5dS0KhkDiOYxKYIiIKDAwkTU1NevHiBV27dk0hMHX79m0yNjZmokv06ZrdTp06kYeHBzPt/Px8Onv2LLVu3Zrs7e3p0qVLtHTpUqqoqKBu3box3VWVcerUKfr999+rna9bty4fPKht/mmCpsqA1OXLl+n27dv0/v17hfNdunRhonft2jVq164d3b59mziOo5YtW9LGjRvJwsKCiIhKS0tpwIABzP4GK1eupG3btlFERATp6elR3759KTo6munnXB5XV1dydXVViZY86vZ5kqdp06bUtGlTWrx4MaWnp9OqVauobdu2tR6kkH+2FBcX04QJE6h///4Kngxr1qyh2bNn16ruP/Hs2TO6fv06WVhYkJWVFTMdJycnunLlCr9Av3fvHunq6vLjN27cYKZfr149GjNmDPXt21fpeF5eHjVq1IiJtjw2Njb04cOHaufLy8upbt26ta63bt06WrNmDd24cYMOHz5M7u7uavMiyc7Opt9//52KioooIyOD6tatS6mpqWRvb08tWrRQyTW8fv1a6fNFlrXHgvLyciooKKgWmCooKGCStSNP06ZN6ebNmwqZyN9++y3TTOTu3bvT6tWrSU9Pj1JSUhS+46qg6uZSVYRCIbMsZXXPpcLDwykyMpJCQ0PpwIEDNH78ePrxxx/pyZMnxHEczZo1i7mH39GjR+nYsWPVAuC2trZ07949ptr29vaUl5dX7X3YvXv3P34uagOJREJHjx6tFpg6evQon7HIkkaNGlGjRo1o4cKFlJGRQcnJyRQcHEzW1ta1vqktn4XMcRytXLmSMjMzlVYXsWLixIn06tUrio2NpXfv3lH79u0pISGBH69bty4lJSUx0RaJRHyCxvv376miokIhYePNmzfMsv+fPXvGzxfq1KlD2traCp95R0dHevDgARNtIvrqMfVfZPDgwfDy8qrmDaCpqcm8Dap8vfS0adOwZ88ehfEff/wRffr0YaKtzppdWWtfIyMj6OrqYv/+/TAwMEBQUBDat28PDQ2Nau1vWWBqasp3TNLR0eFr5Pfu3QsrKyvm+urixo0b8PT0rNaxjHVb665du6JTp0549OgRCgsL0blzZ9jb2+PWrVsAgJKSEpV08Pn7778xd+5cuLi4QCAQoEmTJli+fDnvz1CbbN++/bMPVqjT5+lz2vsq6yBWm7Rp00apJ8P69evRqlUrZro///wz7xn4/v17DBo0SKG1fLdu3Zh5qm3ZsgWHDx+ucXz27NmYPHkyE+3w8HCMHj26xvG8vDxwHMdEW55t27ahadOmOHXqFN/J59SpU2jevDmzLmkyWrdurTaPqc2bN0MikSA6OhpaWlr8c23p0qXo0KEDc/2HDx8iJCSEf55UPVgyZswYGBoaIj4+HtnZ2cjOzkZ8fDyMjIyYdCnz9vbG06dPAVR2QlN1+3p5jxd5jzNVU1RUpPQ+fu3aNQVfGha8fPkSWVlZSEtLQ3p6OrKyspj7KwFAeXk5Zs6ciU6dOmHOnDkAgI0bN8La2hpGRkbo378/ysrKmF6DoaEhv06Rn8NmZ2fD1NSUqXZycjLq1q2LtLQ0SKVSbNy4ETNnzuT/zZrZs2dDS0sLMTExSE1NRWpqKmJiYiCRSDB79mwmmv/0HSssLMTEiRNrXbd169afdQQEBNS69pdAaGgoOnXqhKNHj2Lw4MFo3LgxQkJCUFZWhlevXqFnz54IDg5mom1jY6PgvTt+/HjejxionM8YGxsz0QaAr4EpNaHuttZbt26FtbU1EhMT+XOsA1P5+fkoLy9n9vP/CT8/P8TExODly5eIj4+HlZUVYmJi+PEff/wRvr6+TLQbNmyImTNnAqh8kBsYGGDGjBn8+Pz58+Hl5cVEW55Bgwaha9eueP/+PXR0dFBUVIRbt27B29sbo0aNYqa7dOlSBAYGolevXvxnXsajR49gb2/PTBsAOnXqhNDQUDx8+BA6Ojq4fPkysrOz0bRpUxw5coSZrqmpKc6fP69wbtiwYbCxscGNGzdUFpiS58iRI+jXrx+kUimkUikTXflDPhAoHwxkuWDjOA6XLl3CokWL0KBBAwgEAnh6eiIxMZFfVLFCnQt0GRKJROmi6erVq5BIJMx05Sexs2bNgomJCf744w/cu3cPO3bsQN26dRXue/8WHjx4gOLiYrVoGxgY8EbnhoaGEIlEEAgEfFtt2b8NDQ3Vcn2qwMvLize7ll+snjt3DmZmZsz1w8PD4evri9zcXEilUmRmZiI1NRXOzs7466+/mGp//PgRc+fOhaWlJX+PtbS0xNy5c5nMt8RiMd/AQB2BIXWbn8vw9/fH6tWrq51PTU1lFvz/8OEDRo4cCYlEAo7joKWlBZFIBI7jIJFIMGrUKLx//56J9pdC7969MWjQIADg57AvX75EmzZtVNJAaPny5bCxseG/a1ZWVli5ciVzXRnp6enw9fXl7/e+vr7Mm7moK/j7X+batWtwcnICx3Fwd3fHvXv30KVLF2hqakJTUxMmJiY4c+YME+0uXbpg8eLFNY7/+uuvaNOmDRNt4GtgSi2kpqZCU1MTDRs2hI6ODlJSUmBgYIDo6GhERUVBJBIhIyOD+XXcvXsXbdq0QXBwMB48eMA8MCUQCPDw4UMAgL29PR4/fsxMSxl6enq4fv06gMoHvKamJs6dO8ePX7t2Dfr6+ky0pVIpv4sm6+wgH7C4ceMG8850AFBaWgo/Pz8YGBhAQ0MD1tbWEAqF8Pf3Z7bTlZCQAG1tbcTExKBv377Q0tJCXFwcP84yOCPDyMgI+fn5ACo/B1euXAEAHDhwgGlAUFdXV2nXouHDh8PKygpHjhxh9rv/06KhtLQUy5c
```

Comment:
The rest of the analysis is completely wrong. Do not hesitate to ask if you don't know something 😉